tools/outline_proxy_controller/OutlineProxyControllerConfig.h.in

```diff
@@ -0,0 +1,24 @@
+/**
```

Comment:
Revert?

Can you add the instructions to generate this to [the README?](https://github.com/Jigsaw-Code/outline-client/blob/master/tools/outline_proxy_controller/README.md)?

Reply:
Oh this one is not generated, it is a source code file (ended in `.h.in`), CMake will generate a corresponding `.h` file according to this file. I already remove the generated `.h` file.